po/manifest2po

```diff
@@ -67,14 +67,17 @@ function step() {
         return;
     }
 
-    if (path.basename(filename) != "manifest.json")
+    if (path.basename(filename) != "manifest.json.in")
         return step();
 
     fs.readFile(filename, { encoding: "utf-8"}, function(err, data) {
         if (err)
             fatal(err.message);
 
-        process_manifest(JSON.parse(data));
+        // There are variables which when not substituted can cause JSON.parse to fail
+        // Dummy replace them. None variable is going to be translated anyway
+        safe_data = data.replace(/\@.+?\@/gi, 1);
```

Comment:
This was the reason why the POT should really be generated from the .manifest, not the .in. I. e. what's missing is a dependency of po/POTFILES.manifest.in to all the manifest.json files, so that they are built first. This would have been a cleaner solution, but *shrug*, it's good enough. :-)